javascript/features/friends/friends_commands.js

```diff
@@ -39,7 +39,7 @@ export class FriendsCommands {
             player.sendMessage(Message.FRIENDS_ERROR_FRIEND_NOT_REGISTERED, friendPlayer.name);
             return;
         }
-
+ 
```

Comment:
NIT: I asume this was a mistake?

Reply:
Added a condition to prevent NPCs to be added in player's friend list 
But....... later found out NPCs are considered unregistered so we can't add them friends either :(

Reply:
Yup it was a mistake :(